pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -53,12 +54,13 @@ type Response struct {
 
 type Client struct {
 	*client.Client
-	Request  Request
-	Response Response
-	Error    bool
-	Created  bool
-	server   *httptest.Server
-	handler  *utiltesting.FakeHandler
+	Clientset *release_1_1.Clientset
```

Comment:
Since we are calling AdaptOldClient below, this is not really release 1.1 client as it seems.
It is a clientset based on the client above.
Maybe mention that in the comment?